api/metric/noop.go

```diff
@@ -7,7 +7,15 @@ import (
 )
 
 type NoopProvider struct{}
-type NoopMeter struct{}
+
+type NoopMeter struct {
+	NoopSyncMetricConstructors
+	NoopAsyncMetricConstructors
+}
+
+type NoopSyncMetricConstructors struct{}
```

Comment:
This seems like a hold-over from the interface split, right?

Reply:
Argh. Sorry I've been lazy here, thanks again.

Reply:
NP :)